File: src/programmers/lv0/lv0_120818/Solution.js
function solution(price) {
    price = price >= 500000 ? price*0.8
                            : price >= 300000 ? price*0.9
                                              : price >= 100000 ? price*0.95 
                                                                : price;
    price = Math.floor(price);
    return price;
}

File: src/programmers/lv0/lv0_120818/Solution.java
package programmers.lv0.lv0_120818;

class Solution {
    public int solution(int price) {
        if(price >= 100000 && price<300000){
            price = (int)((double)price*0.95);
        }else if(price >= 300000 && price<500000){
            price = (int)((double)price*0.90);
        }else if(price >= 500000){
            price = (int)((double)price*0.80);
        }
        return price;
    }
}